README.md

```diff
@@ -41,7 +41,7 @@ endpoint.  There is a public REST service running on AWS but the service can
 also be run locally as
 
 ```bash
-python -m gilda.app
+gilda
```

Comment:
One reason I prefer the old version here is that it works both when the package is pip installed, and when one just clones the code and puts it on the PYTHONPATH. I think if there were a CLI (so far there hasn't been a use case where that would have been needed) I think it would make sense to add the entrypoint and document it this way.

Reply:
Hmm I hadn't considered someone would want to clone it and put it on the python path rather than just installing it. I could add a proper CLI around it if you want! I actually made a package that has my common patterns for doing this with a flask app (https://github.com/cthoyt/more_click/)

Reply:
Moved to #42